pkg/collector/collector.go

```diff
@@ -78,19 +78,20 @@ func (b *BaseCollector) Export() error {
 }
 
 // Helper function to collect output of a given kubectl command to a file. Returns kubectl's stderr output if stdout output is empty.
-func (b *BaseCollector) collectKubeResourceToFile(rootPath, fileName string, kubeCmds []string) error {
-	resourceFile := filepath.Join(rootPath, fileName)
+func (b *BaseCollector) AddKubectlOutputToCollectorFiles(rootPath, fileName string, kubeCmds []string) error {
```

Comment:
Hmm I have mixed feelings about this rename. "add" implies it is just adding but this function does actual collection because it also runs the command, creates the file, write to file and then adds the file to collector files.

Reply:
vs the existing AddToCollectorFile which just literally "adds" an already created file to the collector's folder

Reply:
Hmm yeah I see what you mean, I'll change it